stdlib/src/collections/inline_list.mojo

```diff
@@ -20,6 +20,7 @@ from collections import InlineList
 """
 
```

Comment:
I love how clean the changes are in this file with the introduction of `UnsafeMaybeUninitialized` 🙂